Documentation/logging.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Logging
+description: Learn about how you should use logging in your code
+keywords: Contributing
```

Comment:
We use keywords for our search so it's good to add some of the most common keywords when you'd be searching for this sort of information
```suggestion
keywords: Contributing, logging, logs, trace, debug, warning, error
```